pkg/karmadactl/top/top_pods.go

```diff
@@ -0,0 +1,274 @@
+package top
+
+import (
```

Comment:
We will add `top node` in the future?

Reply:
If there are user needs, this feature will be added.